filter_plugins/trellis_filters.py

```diff
@@ -0,0 +1,20 @@
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import re
+
+def reverse_www(host=None):
+    ''' Add or remove the www subdomain '''
+    match = re.match(r'www\.(.+)', host)
+    if match:
+        return match.group(1)
+    return 'www.' + host
```

Comment:
Not sure what idiomatic Python is but seems like this should be under the `else` condition.